package.json

```
@@ -6,7 +6,9 @@
     "sass": "npx sass libs/portal-integration-angular/assets/styles.scss libs/portal-integration-angular/assets/output.css",
     "postinstall": "mkdir -p .nx",
     "build": "nx run-many -t build build-migrations",
-    "build-copy": "npm run build && node copy-build.js"
+    "build-copy": "npm run build && node copy-build.js",
+    "sonar": "node sonar.js",
+    "sonar:all": "node sonar.js"
```

Just from reading the scripts and without knowing the code. I wouldn't know what the difference between these two is and how I should use them. Please add documentation to the README and think about if it makes sense to provide two scripts instead of one. The benefit of two scripts is that you can throw an exception in sonar.js if no folder is provided.